_layouts/default.html

```diff
@@ -6,9 +6,35 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- SHORTCUT ICON -->
     <link rel="shortcut icon" href="https://developer.sketchup.com/sites/developer.sketchup.com/files/Favicon.ico" type="image/vnd.microsoft.icon" />
-    <title>Homepage | SketchUp Developer</title>
-    
+  <!-- TITLE -->
+    <title>SketchUp Extension UX Guidelines</title>
+
+  <!-- GENERIC META PROPERTIES -->
+    <meta name="url"   content="https://sketchup.github.io/sketchup-extension-ux-guidelines/" />
+    <meta name="image" content="https://www.sketchup.com/sites/www.sketchup.com/files/developer/warehouse.png" />
```

Comment:
These files are not stable. They can easily change, and they have in the past. Better to host them with this repo.

https://www.sketchup.com/sites/www.sketchup.com/files/developer/warehouse.png

Reply:
Good point!

Do you have a link to the general SU logo? As of now extensibility doesn't have any icon, and this project isn't a part of the Extension Warehouse.

Reply:
> _These files are not stable. They can easily change, and they have in the past. Better to host them with this repo._

I do not understand this statement. Please be specific. **What files? What URLs change ?**

The idea is that when a URL is pasted in a Discourse forum (anywhere), in Twitter, in FaceBook, in wherever oEmbed or OpenGraph one boxes are supported, that the page referenced has the meta tags that have the properties to populate the onebox or twitter card or whatever.

So a stable URL is needed for pasting into other sites. I do not care **what** it is. As long as it's stable. (Ie, there are numerous links previously posted to the forum that have no redirects and are now "dead links".

> _Do you have a link to the general SU logo?_

I could not find a square application logo in png format. 
All the images I found have a gray "SketchUp" to the right of the logo.

Is there a logo only image embedded within the executable ? 
(Perhaps used for message boxes? or the installer?)

> _As of now extensibility doesn't have any icon, and this project isn't a part of the Extension Warehouse._

I was under the impression that the new blue X logo stood for extensibility (ie extensions) and that EW was generally a part of SketchUp's extensibility.

Reply:
Do you wish to also put the favicon in this repo's "assets" folder ?

Reply:
Logo has been changed to Mark's logo: copied into assets folder.